README.md

```diff
@@ -20,7 +20,7 @@ The application implements the GOV.UK design system. Visit the project website [
 To easily implement the GOV.UK design system, the application uses Nunjucks as a templating language. Visit the website [https://mozilla.github.io/nunjucks](https://mozilla.github.io/nunjucks) for more information and documentation.
 
 ### Creating a new step in the form
-To add a new form step to the application, create an object matching the following schema 
+To add a new form step to the application, create an object matching the following schema
 
```

Comment:
These space updates were implemented automatically by VS Code - sorry about that!!